### ISSUE TITLE
Magento 2.4.0 compatibility

### ISSUE THREAD

**Issue by dvershinin** [integer-net/magento2-session-unblocker#16]
```
    - magento/product-community-edition 2.4.0 requires magento/framework 103.0.0 -> satisfiable by magento/framework[103.0.0].
    - Installation request for magento/product-community-edition 2.4.0 -> satisfiable by magento/product-community-edition[2.4.0].
    - Installation request for integer-net/magento2-session-unblocker ^0.1.4 -> satisfiable by integer-net/magento2-session-unblocker[v0.1.4].
```

**Comment by wigman** [comment]
Is there a hidden question?

**Comment by dvershinin** [comment]
Can this module be used with M 2.4? If so, I guess, [it should be updated](https://github.com/integer-net/magento2-session-unblocker/blob/master/composer.json#L15) to include 103.0.0.

**Comment by wigman** [comment]
Right now we don't use this module in any of our active projects, so I don't know yet. It will take a while until testing this get's priority from our side.
Unless this ticket gets a lot of traction, I'm going to wait for someone else to report back whether it works.

We would gladly accept a pull-request or update the requirements after confirmation that it works.

**Comment by rhoerr** [comment]
Noting interest

**Comment by webhive** [comment]
Same issue 

``` sh
- magento/framework 102.0.6 requires php ~7.1.3||~7.2.0||~7.3.0 -> your PHP version (7.4.13) does not satisfy that requirement.
- integer-net/magento2-session-unblocker v0.1.4 requires magento/framework ^100.1|^101.0|^102.0 -> satisfiable by magento/framework[102.0.6, 102.0.5-p2, 102.0.4-p2, 102.0.5, 102.0.3-p1, 102.0.4, 101.0.11, 102.0.2-p2, 101.0.10,
 102.0.3, 101.0.9, 100.1.18, 102.0.2, 101.0.8, 102.0.1, 100.1.17, 102.0.0, 101.0.7, 100.1.16, 101.0.6, 100.1.15, 100.1.14, 101.0.5, 101.0.4, 100.1.13, 101.0.3, 100.1.12, 101.0.2, 100.1.11, 100.1.10, 101.0.1, 101.0.0, 100.1.9, 100.
1.8, 100.1.7, 100.1.6, 100.1.5, 100.1.4, 100.1.3, 100.1.2, 100.1.1, 100.1.0].
- Installation request for integer-net/magento2-session-unblocker ^0.1.4 -> satisfiable by integer-net/magento2-session-unblocker[v0.1.4].
```

It could be great to include 103.0.0

**Comment by wigman** [comment]
@webhive @rhoerr @dvershinin please test https://github.com/integer-net/magento2-session-unblocker/tree/16-magento-2.4

If it works, I'll merge it.

**Comment by webhive** [comment]
Unfortunately do not need it anymore as resolved my issues different way.
But when installed it I  had no issues.
Really not sure if it was even working as found no changes.

**Comment by Maksold** [comment]
> Unfortunately do not need it anymore as resolved my issues different way.
> But when installed it I had no issues.
> Really not sure if it was even working as found no changes.

Could you please share how you resolved these issues?

**Comment by bselimb** [comment]
I'll try and let you know, this extension should be included in to the core package because I'm seeing session lock issues on all my projects with pretty much standard redis configuration.

**Comment by treeofinformation** [comment]
Any chance this might be revisited? I'm interested in trying the module on one of my 2.4 instances.

**Comment by HenKun** [comment]
<​item​ ​name​=​"​reviewSession​"​ ​xsi​:​type​=​"​object​"​>Magento\Review\Model\Session</​item​>is only available in frontend. So you have to put the di.xml into frontend/di.xml
Besides that it runs without errors but I couldn't see any performance improvement

**Comment by schmengler** [comment]
Thanks @HenKun - if anyone wants to make a PR for this, I'll merge it. We don't currently use the module ourselves

**Comment by nntoan** [comment]
Hi @schmengler , just curiosity can you share how you guys handle the session unblocking without the module? Is that just the configuration of Redis?

**Comment by EvanRijn** [comment]
@schmengler 

> Thanks @HenKun - if anyone wants to make a PR for this, I'll merge it. We don't currently use the module ourselves

How do fix it now? Disable redis locking ?

**Comment by schmengler** [comment]
@nntoan @EvanRijn No, we just don't have projects anymore where parallel requests are a noteworthy problem, and when there's no need for the module, I wouldn't install it

**Comment by schmengler** [comment]
Closed via #17 => [Release 0.1.5](https://github.com/integer-net/magento2-session-unblocker/releases/tag/v0.1.5)

**Comment by JMLucas96** [comment]
Hi, we have noticed that in m2.4.3-p3 sometimes a customer session is recovered in another customer_id, is it possible? We are going to remove that module if there's no needed anymore. 
Keep in touch. 👍
